sdk/tests/conformance2/rendering/element-index-uint.html

```diff
@@ -309,13 +308,10 @@
     var indices = new Uint32Array([ 10000, 0, 1, 2, 3, 10000 ]);
     gl.bufferData(gl.ELEMENT_ARRAY_BUFFER, indices, drawType);
     wtu.shouldGenerateGLError(gl, gl.NO_ERROR, "gl.drawElements(gl.TRIANGLE_STRIP, 4, gl.UNSIGNED_INT, 4)");
-    wtu.shouldGenerateGLError(gl, gl.INVALID_OPERATION, "gl.drawElements(gl.TRIANGLE_STRIP, 4, gl.UNSIGNED_INT, 0)");
-    wtu.shouldGenerateGLError(gl, gl.INVALID_OPERATION, "gl.drawElements(gl.TRIANGLE_STRIP, 4, gl.UNSIGNED_INT, 8)");
-    indices[0] = 2;
-    indices[5] = 1;
+    wtu.shouldGenerateGLError(gl, gl.INVALID_OPERATION, "gl.drawElements(gl.TRIANGLE_STRIP, 4, gl.UNSIGNED_INT, 28)");
+    indices = new Uint32Array([ 10000, 0, 1, 2, 3, 10000, 2, 1]);
```

Comment:
This was part of runCopiesIndicesTests. It looks like there was once a bug where the ArrayBuffer holding the indices was being held by reference, instead of copied.